docs/cas-server-documentation/installation/Configuration-Properties.md

```diff
@@ -879,12 +879,16 @@ to be a JSON map as such:
 
 ### Ruby/Python/Javascript/Groovy
 
-Similar to the Groovy option but more versatile, this option takes advantage of Java's native scripting API to invoke Groovy, Python or Javascript scripting engines to compile a pre-defined script o resolve attributes. The following settings are relevant:
+Similar to the Groovy option but more versatile, this option takes advantage of Java's native scripting API to invoke Groovy, Python or Javascript scripting engines to compile a pre-defined script to resolve attributes. The following settings are relevant:
 
 ```properties
 # cas.authn.attributeRepository.script[0].location=file:/etc/cas/script.groovy
 # cas.authn.attributeRepository.script[0].order=0
 # cas.authn.attributeRepository.script[0].caseInsensitive=false
+# The following property is optional and may contain any engine supported by Java's ScriptEngineManager
```

Comment:
Just a cautionary advisory: try to move the comments to the code as javadoc, if needed. This will allow us to auto-gen properties and notes later on based on the cas config metadata as part of the overlay generation processor separately, as part of the work @mindblender is doing.